app/src/pages/admin/projectsPage/projects/projects.jsx

```diff
@@ -61,14 +74,19 @@ export class Projects extends Component {
 
         {viewMode === TABLE_VIEW && <ProjectsGrid />}
 
-        <PaginationToolbar
-          activePage={activePage}
-          itemCount={itemCount}
-          pageCount={pageCount}
-          pageSize={pageSize}
-          onChangePage={onChangePage}
-          onChangePageSize={onChangePageSize}
-        />
+        {!!pageCount &&
```

Comment:
By my own opinion, that should be a part of grid component

Reply:
Good point